### ISSUE TITLE
Enhancement: Minimum votes for update

### ISSUE THREAD

**Issue by jobrien2001** [mynttt/UpdateTool#118]
Hello,

It would be good to have an environmental variable to set a minimum amount of votes for an update to happen.

The problem is a movie can have a 10 rating but only few votes... making it an unreliable rating.

There are too many new movies with high rating, making sorting movies by rating useless.

Thanks

**Comment by mynttt** [comment]
I agree that this is a useful feature, searching IMDB with minimum votes < 1000 yields some really bad results.

One question would be how are ratings below the threshold handled that have already been updated? Should they be reverted to 0 as a signal that they're categorized as too low for their votes to have any significance? Should that be an additional option?

**Comment by jobrien2001** [comment]
Im not sure. 

Some of the data from the file in updatetool seems to be off from the actual rating, maybe its cached and not refreshed as often.

Maybe set rating to null and trigger a refresh so it gets the actual rating from the default agent?  Im not sure if the agents handle this problem already

For this an env variable would be needed for a plex token and another for a host to send a curl request.

**Comment by mynttt** [comment]
@jobrien2001 

Can you provide examples for data that is wrong? Data is sourced from the daily updated IMDB data sets or scraped from their website (then cached for 7 days); having completly off data would indicate that something in the ID matching process is wrong and would mean that the tool is possibly having a bug.

**Comment by jobrien2001** [comment]
Hello, 

Im not sure the data is wrong.  Since you say its cached for 7 days, maybe at some point earlier it was right, unreliable because of the low vote count, but right.

I see the way the data is presented.

Perhaps your suggestion is the best solution, set 0 or NULL(havent looked at db) to any record below a determined amount of votes.

An env variable would be great so the user can set that number to their liking.

**Comment by robertybob** [comment]
> An env variable would be great so the user can set that number to their liking.


This ^^ 

If you’re unlucky enough to be a fan of obscure shows or movies then it seems a bit unfair to be punished if <100 users voted on them, for example.
